pom.xml

```diff
@@ -240,7 +240,7 @@
         <version.io.opentracing.jaxrs2>0.1.7</version.io.opentracing.jaxrs2>
         <version.io.opentracing.tracerresolver>0.1.5</version.io.opentracing.tracerresolver>
         <version.io.opentracing.servlet>0.1.0</version.io.opentracing.servlet>
-        <version.io.reactivex.rxjava>2.1.16</version.io.reactivex.rxjava>
+        <version.io.reactivex.rxjava>2.2.0</version.io.reactivex.rxjava>
```

Comment:
This is also being used by org.jboss.resteasy.resteasy-rxjava2 module, @asoldano is this upgrade ok for resteasy?

Reply:
Actually - this upgrade was meant for Infinispan 9.4.x, not this release.  Correcting...